source/VexiiRiscv/Introduction/index.rst

```diff
@@ -1,20 +1,19 @@
 Introduction
 ============
 
-
-About VexiiRiscv
-------------------------------
-
-VexiiRiscv is a from scratch second iteration of VexRiscv, with the following goals : 
-
-- RISCV 32/64 bits IMAFDC
-- Could start around as small as VexRiscv, but could scale further in performance
-- Optional late-alu
-- Optional multi issue
-- Optional multi threading
-- Providing a cleaner implementation, getting ride of the technical debt, especially the frontend
-- Proper branch prediction
-- ...
+VexiiRiscv is the next generation of VexRiscv, enabling things which are not possible with the current VexRiscv framework.  The new VexiiRiscv framework: 
```

Comment:
> next generation

The wording isn't great it think, because it is time sensitive. While "second iteration" is timeless.

Also overall it lose the meaning that VexiiRiscv is a from scratch implementation

> framework

I would say this should be in the bullet point, not as an introduction text